Cesium3DTiles/src/BingMapsRasterOverlay.cpp

```diff
@@ -352,73 +439,15 @@ BingMapsRasterOverlay::createTileProvider(
               return nullptr;
             }
 
-            rapidjson::Document response;
-            response.Parse(
-                reinterpret_cast<const char*>(pResponse->data().data()),
-                pResponse->data().size());
+            const std::byte* responseBuffer = pResponse->data().data();
+            size_t responseSize = pResponse->data().size();
 
-            if (response.HasParseError()) {
-              SPDLOG_LOGGER_ERROR(
-                  pLogger,
-                  "Error when parsing Bing Maps imagery metadata, error code "
-                  "{} at byte offset {}",
-                  response.GetParseError(),
-                  response.GetErrorOffset());
-              return nullptr;
-            }
-
-            rapidjson::Value* pResource =
-                rapidjson::Pointer("/resourceSets/0/resources/0").Get(response);
-            if (!pResource) {
-              SPDLOG_LOGGER_ERROR(
-                  pLogger,
-                  "Resources were not found in the Bing Maps imagery metadata "
-                  "response.");
-              return nullptr;
-            }
-
-            uint32_t width =
-                JsonHelpers::getUint32OrDefault(*pResource, "imageWidth", 256U);
-            uint32_t height = JsonHelpers::getUint32OrDefault(
-                *pResource,
-                "imageHeight",
-                256U);
-            uint32_t maximumLevel =
-                JsonHelpers::getUint32OrDefault(*pResource, "zoomMax", 30U);
-
-            std::vector<std::string> subdomains =
-                JsonHelpers::getStrings(*pResource, "imageUrlSubdomains");
-            std::string urlTemplate = JsonHelpers::getStringOrDefault(
-                *pResource,
-                "imageUrl",
-                std::string());
-            if (urlTemplate.empty()) {
-              SPDLOG_LOGGER_ERROR(
-                  pLogger,
-                  "Bing Maps tile imageUrl is missing or empty.");
-              return nullptr;
-            }
+            SessionCacheItem& item = sessionCache.emplace_back();
+            item.metadataUrl = metadataUrl;
+            item.responseData.resize(responseSize);
+            std::memcpy(item.responseData.data(), responseBuffer, responseSize);
```

Comment:
I think these two lines can be replaced with:
```
item.responseData = pResponse->data();
```

Worst case, this will definitely work:

```
item.responseData = std::vector<std::byte>(pResponse->data().begin(), pResponse->data().end());
```